controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -2144,6 +2201,104 @@ int cli_bml::get_dcs_scan_results(const std::string &radio_mac, uint32_t max_res
+{
+    char client_list[256];
+    unsigned int client_list_size;
+
+    int ret = bml_client_get_client_list(ctx, client_list, &client_list_size);
+    auto client_list_vec =
+        beerocks::string_utils::str_split(std::string(client_list, client_list_size), ',');
+
+    std::cout << "client list:" << std::endl;
+    for (const auto &client : client_list_vec) {
+        std::cout << "- " << client << std::endl;
+    }
+
+    printBmlReturnVals("bml_client_get_client_list", ret);
+
+    return 0;
+}
+
+/**
+ * Set specific client according to MAC.
+ * 
+ * @param [in] sta_mac MAC address of requested client.
+ * @param [in] selected_bands comma-seperated selected bands.
+ * @param [in] stay_on_initial_radio Whather to stay on initial radio or not.
+ * @param [in] stay_on_selected_device Whather to stay on selected device or not.
+ * @return 0 on success.
+ */
+int cli_bml::client_set_client(const std::string &sta_mac, int8_t selected_bands,
+                               bool stay_on_initial_radio, bool stay_on_selected_device)
+{
+    BML_CLIENT_CONFIG cfg;
+
+    cfg.selected_bands          = selected_bands;
+    cfg.stay_on_initial_radio   = stay_on_initial_radio ? 1 : 0;
+    cfg.stay_on_selected_device = stay_on_selected_device ? 1 : 0;
+
+    int ret = bml_client_set_client(ctx, sta_mac.c_str(), &cfg);
+
+    printBmlReturnVals("bml_client_set_client", ret);
+
+    return 0;
+}
+
+/**
+ * get specific client according to MAC.
+ * 
+ * @param [in] sta_mac MAC address of requested client
+ * 
+ * @return 0 on success.
+ */
+int cli_bml::client_get_client(const std::string &sta_mac)
+{
+    BML_CLIENT client;
+    int ret = bml_client_get_client(ctx, sta_mac.c_str(), &client);
+
+    if (ret == BML_RET_OK) {
+        auto client_bool_print = [](int8_t val) -> std::string {
+            if (val == BML_PARAMETER_NOT_CONFIGURED)
+                return "Not configured";
+            return val ? "True" : "False";
+        };
+        auto client_selected_bands_print = [](int8_t val) -> std::string {
+            std::string ret = "";
+            if (val == BML_CLIENT_SELECTED_BANDS_DISABLED)
+                return "Disabled";
+            if (val & BML_CLIENT_SELECTED_BANDS_24)
+                ret += "2.4 Ghz,";
+            if (val & BML_CLIENT_SELECTED_BANDS_5G)
+                ret += "5 Ghz,";
+            if (val & BML_CLIENT_SELECTED_BANDS_6G)
+                ret += "6 Ghz,";
+            if (val & BML_CLIENT_SELECTED_BANDS_60)
+                ret += "60 Ghz,";
+            return ret;
+        };
+        std::cout << "client: " << client.sta_mac << std::endl
+                  << " timestamp_sec: " << client.timestamp_sec << std::endl
+                  << " stay_on_initial_radio: " << client_bool_print(client.stay_on_initial_radio)
+                  << std::endl
+                  << " stay_on_selected_device: "
+                  << client_selected_bands_print(client.selected_bands) << std::endl
```

Comment:
you have an error here in the print
reorganize it

Reply:
Fixed it